test_optimize_algorithm.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from command_buffer import CommandBuffer
+from buffer_driver import BufferDriver
+from nand import Nand
+
+FILE_PATH = "test_ssd_nand.txt"
+LBA_LENGTH = 100
+DEFAULT_VALUE = "0x00000000"
+FIRST_ADDRESS = 0
+LAST_ADDRESS = LBA_LENGTH - 1
+
+
+@pytest.fixture
+def command_buffer():
+    nand = Nand(LBA_LENGTH, FILE_PATH)
+    return CommandBuffer(nand)
+
+
+def test_ignore_write(command_buffer):
+    buffers = [('E', 0, 3), ('W', 0, '0x12345678'), ('W', 1, '0x11111111'), ('E', 0, 5)]
+    new_buffers = command_buffer.ignore_write(buffers)
+    assert new_buffers == [('E', 0, 3), ('E', 0, 5)]
```

Comment:
여기는 그냥 한줄로 합쳐도 좋을것 같네요.
```suggestion
assert command_buffer.ignore_write(buffers) == [('E', 0, 3), ('E', 0, 5)] 
```